ontopy/ontology.py

```diff
@@ -191,36 +213,29 @@ def __objclass__(self):
         pass
 
     def __hash__(self):
-        """ Returns hash based on base_iri
+        """Returns a hash based on base_iri.
         This is done to keep Ontology hashable when defining __eq__.
         """
         return hash(self.base_iri)
 
     def __eq__(self, other):
-        """Checks if this ontology is equal to other.
+        """Checks if this ontology is equal to `other`.
 
-        Equality of all triples obtained from self.get_unabbreviated_triples(),
-        i.e. blank nodes are not distinguished, but relations
-        to blank nodes are included.
+        This function compares the result of
+        ``set(self.get_unabbreviated_triples(bnode='_:b'))``,
+        i.e. blank nodes are not distinguished, but relations to blank
+        nodes are included.
         """
-        return set(self.get_unabbreviated_triples()) == set(
-            other.get_unabbreviated_triples()
+        return set(self.get_unabbreviated_triples(bnode='_:b')) == set(
+            other.get_unabbreviated_triples(bnode='_:b')
         )
 
-    def get_unabbreviated_triples(self):
-        """ Returns all triples unabbreviated
-        """
-        def _unabbreviate(i):
-            if isinstance(i, int):
-                if i >= 0:
-                    return self._unabbreviate(i)
-                return "_:"  # blank nodes are given random neg. storid
-            return i
+    def get_unabbreviated_triples(self, bnode=None):
+        """ Returns all triples unabbreviated.
 
-        for subject, predicate, obj in self.get_triples():
-            yield (_unabbreviate(subject),
-                   _unabbreviate(predicate),
-                   _unabbreviate(obj))
+        If `bnode` is given, it will be used to represent blank nodes.
+        """
+        return World.get_unabbreviated_triples(self, bnode)
```

Comment:
```suggestion
        return self.world.get_unabbreviated_triples(self, bnode)
```
Shouldn't this be using the method from it's stored world ontology, or?